src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -467,6 +464,14 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
       ResourceName="XA1023_dotnet"
       Condition="  '$(AndroidDexTool)' == 'dx' and '$(UsingAndroidNETSdk)' == 'true' "
   />
+  <AndroidWarning Code="XA1026"
+      ResourceName="XA1026"
+      Condition=" '$(AndroidUseAapt2)' == 'false' and '$(UsingAndroidNETSdk)' != 'true' "
+  />
+  <AndroidError Code="XA1026"
+      ResourceName="XA1026_dotnet"
+      Condition=" '$(AndroidUseAapt2)' == 'false' and '$(UsingAndroidNETSdk)' == 'true' "
+  />
```

Comment:
Should there be a condition on a `CallTarget` in here , for the `Aapt` target.

Reply:
My plan was to not import or ship `Xamarin.Android.Aapt.targets` at all:

https://github.com/xamarin/xamarin-android/blob/02e86b5a6bbd6f938c16cf30bc709543565bb34e/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L364-L365

However this is wrong ☝️ , it needs to be checking `$(UsingAndroidNETSdk)` instead.

This should also work:

https://github.com/xamarin/xamarin-android/blob/91b66985704f1592b8c55819f7ff48d5e7353900/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L1160-L1161

Reply:
I changed this to:
```xml
<Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Aapt.targets"  Condition=" '$(AndroidUseAapt2)' != 'true' and '$(UsingAndroidNETSdk)' != 'true' " />
<Import Project="$(MSBuildThisFileDirectory)Xamarin.Android.Aapt2.targets" Condition=" '$(AndroidUseAapt2)' == 'true' or '$(UsingAndroidNETSdk)' == 'true' " />
```
A couple quick tests, things seem to work? Will need to check CI results.